From fc1eb921bc308e332c044c8a540e93f0e6e5e53d Mon Sep 17 00:00:00 2001
From: George Burgess IV <gbiv@google.com>
Date: Fri, 6 Aug 2021 22:57:06 -0700
Subject: [PATCH] update protobuf-codegen for the rust uprev

The Rust 1.54 uprev makes the use of `#![rustfmt::skip]` an error:
https://github.com/rust-lang/rust/issues/64266. Hence, we need to remove
uses of that.

It seems that
https://github.com/stepancheg/rust-protobuf/commit/b06075c2e887da3dd00adf9d7b376af47c472f0c
fixes the uses of this in protobuf-y packages.

BUG=b:194823966
TEST=emerge-zork-floss ${package_that_was_broken}

Change-Id: Ic564af8c78436ed376937fc276bbcdd94a8b0fe9
---
 src/code_writer.rs             | 2 +-
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/src/code_writer.rs b/src/code_writer.rs
index 8b53bc6..f6d4c07 100644
--- a/src/code_writer.rs
+++ b/src/code_writer.rs
@@ -58,7 +58,7 @@ impl<'a> CodeWriter<'a> {
         self.write_line("#![allow(clippy::all)]");
         self.write_line("");
         self.write_line("#![allow(unused_attributes)]");
-        self.write_line("#![rustfmt::skip]");
+        self.write_line("#![cfg_attr(rustfmt, rustfmt::skip)]");
         self.write_line("");
         self.write_line("#![allow(box_pointers)]");
         self.write_line("#![allow(dead_code)]");
